package.json

```diff
@@ -33,6 +34,7 @@
     "@angular-devkit/build-angular": "^16.0.4",
     "@angular/cli": "^16.0.4",
     "@angular/compiler-cli": "^16.0.4",
+    "@ngneat/spectator": "^15.0.1",
```

Comment:
Added @ngneat/spectator because I could not write test using ngMocks for a specific case